### PR TITLE
Update murmurhash package 

### PR DESCRIPTION
**Pull request by n3omaster**
Be more compatible with PHP 8

### CODE REVIEW

**Comment by pnx** [comment]
Can this be merged? i really need php 8 support.

**Comment by ExpDev07** [comment]
Merge this please and create a release @Bit-Wasp @rubensayshi @afk11

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "bitwasp/bitcoin",
+  "name": "n3omaster/bitcoin-php",
```
Comment (rubensayshi) [inline composer.json:2]: accidental name change

File: composer.json [hunk 2]
```diff
@@ -23,10 +23,10 @@
     }
   },
   "require": {
-    "php-64bit": ">=7.0",
+    "php-64bit": "^7.0|^8.0",
```
Comment (rubensayshi) [inline composer.json:26]: can you also add `0.8` to the `.travis.yml` versions list?

**Comment by ndirpaya** [comment]
> 
Has Php 8 support been added yet?

**Comment by ndirpaya** [comment]
> 
Has Php 8 support been added yet?

**Comment by ExpDev07** [comment]
No, seems like this package is inactive. Perhaps we should create a fork and continue maintaining? @ndirpaya

**Comment by ndirpaya** [comment]
> No, seems like this package is inactive. Perhaps we should create a fork and continue maintaining? @ndirpaya

Someone has forked it already, and it works! https://github.com/n3omaster/bitcoin-php 
Check it out and let me know.

**Comment by ndirpaya** [comment]
> No, seems like this package is inactive. Perhaps we should create a fork and continue maintaining? @ndirpaya

@ExpDev07  Someone has forked it already, and it works! https://github.com/n3omaster/bitcoin-php
Check it out and let me know.

**Pull request closed without merging**